src/metrology_apis/__init__.py

```diff
@@ -36,5 +36,27 @@ def value(self) -> V: ...
     def unit(self) -> U: ...
 
     ### Dunder Methods
-
     def __eq__[B](self: "Quantity[V, U]", other: "Quantity[op.CanEq[V, B], U]", /) -> B: ...
+    def __ne__[B](self: "Quantity[V, U]", other: "Quantity[op.CanEq[V, B], U]", /) -> B: ...
+    def __lt__[B](self: "Quantity[V, U]", other: "Quantity[op.CanOrd[V, B], U]", /) -> B: ...
+    def __le__[B](self: "Quantity[V, U]", other: "Quantity[op.CanOrd[V, B], U]", /) -> B: ...
+    def __gt__[B](self: "Quantity[V, U]", other: "Quantity[op.CanOrd[V, B], U]", /) -> B: ...
+    def __ge__[B](self: "Quantity[V, U]", other: "Quantity[op.CanOrd[V, B], U]", /) -> B: ...
+
+    def __pos__[R](self: "Quantity[op.CanPos[R], U]") -> "Quantity[R, U]": ...
```

Comment:
When would R in `__pos__` ever not be V? I can't think of any examples. 🧐

Reply:
Maybe in SymPy?

Reply:
```pycon
>>> +True
1
```